src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -24,20 +25,30 @@ public class ResourceManagerStringLocalizerFactory : IStringLocalizerFactory
         private readonly ConcurrentDictionary<string, ResourceManagerStringLocalizer> _localizerCache =
             new ConcurrentDictionary<string, ResourceManagerStringLocalizer>();
         private readonly string _resourcesRelativePath;
+        private readonly ILoggerFactory _loggerFactory;
 
         /// <summary>
         /// Creates a new <see cref="ResourceManagerStringLocalizer"/>.
         /// </summary>
         /// <param name="localizationOptions">The <see cref="IOptions{LocalizationOptions}"/>.</param>
+        /// <param name="loggerFactory">The <see cref="ILoggerFactory"/> to use in creating <see cref="ResourceManagerStringLocalizer"/>'s.</param>
```

Comment:
The `<see cref="ILoggerFactory"/>`.~~to use in creating `<see cref="ResourceManagerStringLocalizer"/>`'s.~~</param>